Makefile

```diff
@@ -60,3 +60,9 @@ misspell:
 		go get -u github.com/client9/misspell/cmd/misspell; \
 	fi
 	misspell -w $(GOFILES)
+
+.PHONY: tools
+tools:
+	go install github.com/golang/lint/golint; \
```

Comment:
change golint path to `golang.org/x/lint/golint`

Reply:
thanks, done.